### ISSUE TITLE
Where are the tutorial sources?

### ISSUE THREAD

**Issue by adswa** [realpython/data-version-control#5]
Sorry for posting into the wrong repository - I'm searching for the sources of the [Tutorial](https://realpython.com/python-data-version-control/) that this repository provides the materials for - I'd like to fix a (tiny) bug. I couldn't find it yet, though, do you have a pointer for me? Thanks in advance :)

**Comment by adswa** [comment]
ping @KristijanIvancic 
(closed, because its unrelated to the repo and has been open without activity for two weeks now, but pinged nevertheless, in case the initial issue notification slipped through)

**Comment by kivancic-ut** [comment]
Hi @adswa! Apologies, I didn't get the notification for your original issue. This repo represents the materials for the tutorial you linked. What is the bug you wanted to fix? 🙂

**Comment by adswa** [comment]
Thanks for getting back. :) 
 > What is the bug you wanted to fix?

Very minor stuff - I haven't used DVC before, and executed your complete tutorial, and just noted things that caught me off guard as a new user. If the sources would be open, I would have quickly PRed the changes, they aren't really meaningful. One was: The storage remote is added with a relative path, but the config file entry shows an absolute path. I added my remote from a relative path as shown, and was confused by the mismatching representation of the resulting relative path in the config file (which for relative paths contains a relative path from the config file, not project root, to the remote and looks wrong at first sight). The [docs](https://dvc.org/doc/command-reference/remote/add) recommend using an absolute path:

> Using an absolute path (recommended)

Another one is a minor typo in a code cell command: There was a branch called ``random_forest`` but its local changes are pushed to ``origin random-forest``, which fails when copying the code exactly.

I think I also spotted a few typos, but lost my notes on that between now and opening this issue. Hope none of this comes of as "nit-picky", I just usually try to PR fixes whenever I spot such things.

Cheers,
adina
